Docs- Update info on key rotation (#23274)
hashicorp/vault: website/content/docs/configuration/seal/pkcs11.mdx

the key value, generate a new key in a different key label in the HSM and update Vault's
configuration with the new key label value. Restart your vault instance to pick up the new key
label and all new encryption operations will use the updated key label. Old keys must not be disabled

> or deleted and are used to decrypt older data. If old keys must be disabled or deleted, then [seal-rewrap](/vault/api-docs/system/sealwrap-rewrap#start-a-seal-rewrap-process) needs to be performed so that the vault data encryped by old key can be decrypted using the new key. 

Reviewer: ```suggestion
or deleted and are used to decrypt older data. To disable or delete old keys, Vault needs to
 perform [seal-rewrap](/vault/api-docs/system/sealwrap-rewrap#start-a-seal-rewrap-process)
 so  that data encrypted by the old key can be decrypted using the new key. 
```
Style correction: write in active voice
